LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIStrategyMapItem_AlienNetworkComponent_LW.uc

```diff
@@ -0,0 +1,347 @@
+//---------------------------------------------------------------------------------------
+//  FILE:    UIStrategyMapItem_AlienNetworkComponent_LW
+//  AUTHOR:  Joey Martinez -- 05/06/2019
+//  PURPOSE: This file represents an alien network spot on the StrategyMap for Long War of the Chosen.
+//--------------------------------------------------------------------------------------- 
+
+class UIStrategyMapItem_AlienNetworkComponent_LW extends UIStrategyMapItem_AlienNetworkComponent;
+
+var UIScanButton ScanButton;  // used for missions being infiltrated
+var UIText InfilPct;
+var UIText InfilLabel;
+
+var UIImage SquadImage; // for cases where there is an infiltrating squad
+var string CachedImageName;
+
+//var UIStrategyMap_MissionIcon_LW MissionSiteIcon; // used for pending missions not yet infiltrated 

```

Comment:
Is this needed or not? If it's not, then remove it. It's confusing to have commented out code without an explanation as to why it's commented out and whether it may be uncommented at a later date.

Reply:
done